.github/workflows/stale.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Close Stale PRs'
+on:
+  schedule:
+    - cron: '32 11 * * 1-5'  # 11:32 UTC / 07:32 EDT / 06:32 EST from Mon-Fri.
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30  # Don't wait 6 hours to cancel workflow if it hangs.
+    steps:
+      - uses: actions/stale@v5
+        with:
+          stale-pr-label: 'stale'
```

Comment:
```suggestion
          stale-issue-label: 'stale'
```

Reply:
Good catch, fixed in https://github.com/akuity/kargo/pull/908/commits/7505c22e5eca20ba86829b3f85fb5608a94fb6e2